### PR TITLE
Hannah's Changes

### PR DESCRIPTION
**Pull request by hbruno1304**

### CODE REVIEW

**Review by STARBREED7 — APPROVED** [review]
WHAT HOMIE

**Review by TSunday84 — APPROVED** [review]

File: Hannahs.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+    <!-- Bootstrap CSS -->
+    <link href="Content/bootstrap.min.css" rel="stylesheet" crossorigin="anonymous">
+
+    <title>Hello, world!</title>
+</head>
+<body>
+    <div class="container-fluid">
+        <div class="row">
+            <img src="img/french-bulldog-puppy-food.jpeg" class="img-fluid mx-auto"/>
```
Comment (sschimanski-sh) [inline Hannahs.html]: Hey, might want to make the image name.

**Review by sschimanski-sh — APPROVED** [review]

**Review by Jenna-D-H — APPROVED** [review]
Good job.

**Review by TSunday84 — APPROVED** [review]

**Review by STARBREED7 — APPROVED** [review]
YOUR AWESOME

**Review by STARBREED7 — APPROVED** [review]
YOU ROCK

**Pull request merged**